include/ppx/geometry.h

```diff
@@ -157,6 +170,11 @@ class Geometry
         char*       GetData() { return DataPtr(mData); }
         const char* GetData() const { return DataPtr(mData); }
         uint32_t    GetElementCount() const;
+        uint32_t    GetSizeOfData() const;
```

Comment:
Should be `GetDataSize()` to match convention elsewhere.

Reply:
Done